OsmAnd/src/net/osmand/plus/plugins/development/OsmandDevelopmentPlugin.java

```diff
@@ -219,13 +227,22 @@ public DashFragmentData getCardFragment() {
 		return DashSimulateFragment.FRAGMENT_DATA;
 	}
 
+	@Override
+	public boolean init(@NonNull OsmandApplication app, @Nullable Activity activity) {
+		super.init(app, activity);
+		fpsStatsEnabled = true;
+		fpsStatsCollector();
```

Comment:
initStatsCollector ?

Reply:
renamed to avgStatsCollector (this is basically not init, rather collect method)